modules/iam-role-for-service-accounts-eks/policies.tf

```diff
@@ -284,7 +284,7 @@ data "aws_iam_policy_document" "ebs_csi" {
         "kms:DescribeKey",
       ]
 
-      resources = var.ebs_csi_kms_cmk_ids
+      resources = [for s in var.ebs_csi_kms_cmk_ids : replace(s, "arn:aws:", "arn:${local.partition}:")]
```

Comment:
same for all of these - we don't modify the variables users provide us, they should provide the correct ARNs